src/UI/package.json

```diff
@@ -23,6 +22,7 @@
     "@mui/material": "^5.9.3",
     "@reduxjs/toolkit": "^1.8.3",
     "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "axios": "^0.27.2",
```

Comment:
again, not a thing for you to look into - but I need to sort out why we're not using a more up-to-date version of this library. Axios should be at least above version 1 now (since the last 5 months).